lib/chewy/rake_helper.rb

```diff
@@ -303,7 +303,7 @@ def human_duration(seconds)
 
       def reset_one(index, output, parallel: false)
         output.puts "Resetting #{index}"
-        index.reset!((Time.now.to_f * 1000).round, parallel: parallel)
+        index.reset!((Time.now.to_f * 1000).round, parallel: parallel, progressbar: ENV['PROGRESSBAR'])
```

Comment:
It is nice to check content of progressbar env, because if someone pass `PROGRESSBAR=false`, here we would interpret this value `"false"` as true because it is just a string.

Also I think that `PROGRESS` might be a better shorter namefor this variable, but I have no stronger opinions about it, keep yours it if you prefer it.